```
cd /d C:\projects\neovim\build\src\nvim && C:\msys64\mingw64\bin\cmake.exe -E make_directory C:/projects/neovim/build/windows_runtime_deps
cd /d C:\projects\neovim\build\src\nvim && C:\msys64\mingw64\bin\cmake.exe -DCMAKE_PREFIX_PATH=C:/projects/neovim/.deps/usr -DBINARY="C:/projects/neovim/.deps/usr/bin/swiss.exe" -DDST=C:/projects/neovim/build/windows_runtime_deps -P C:/projects/neovim/cmake/WindowsDllCopy.cmake
CMake Error at C:/msys64/mingw64/share/cmake-3.4/Modules/GetPrerequisites.cmake:798 (message):
  C:/Program Files (x86)/Microsoft Visual Studio 9.0/VC/bin/dumpbin.exe
  failed: 1000
  
  Microsoft (R) COFF/PE Dumper Version 9.00.30729.01
  Copyright (C) Microsoft Corporation.  All rights reserved.
  
  
  Dump of file C:/projects/neovim/.deps/usr/bin/swiss.exe
  
  File Type: EXECUTABLE IMAGE
  
  LINK : fatal error LNK1000: Internal error during DumpSections
  
    Version 9.00.30729.01
  
    ExceptionCode            = C0000005
    ExceptionFlags           = 00000000
    ExceptionAddress         = 009771A0 (00950000) "C:\Program Files (x86)\Microsoft Visual Studio 9.0\VC\bin\link.exe"
    NumberParameters         = 00000002
    ExceptionInformation[ 0] = 00000000
    ExceptionInformation[ 1] = 00000004
  
  CONTEXT:
    Eax    = 40000040  Esp    = 001BDEF0
    Ebx    = 028B3918  Ebp    = 001BDF18
    Ecx    = 00000004  Esi    = 00000004
    Edx    = 0095405C  Edi    = 0000014C
    Eip    = 009771A0  EFlags = 00010246
    SegCs  = 00000023  SegDs  = 0000002B
    SegSs  = 0000002B  SegEs  = 0000002B
    SegFs  = 00000053  SegGs  = 0000002B
    Dr0    = 00000000  Dr3    = 00000000
    Dr1    = 00000000  Dr6    = 00000000
    Dr2    = 00000000  Dr7    = 00000000
Call Stack (most recent call first):
  C:/projects/neovim/cmake/WindowsDllCopy.cmake:19 (get_prerequisites)
  
src\nvim\CMakeFiles\external_swiss.exe.dir\build.make:56: recipe for target 'src/nvim/CMakeFiles/external_swiss.exe' failed
mingw32-make[2]: *** [src/nvim/CMakeFiles/external_swiss.exe] Error 1
mingw32-make[2]: Leaving directory 'C:/projects/neovim/build'
CMakeFiles\Makefile2:394: recipe for target 'src/nvim/CMakeFiles/external_swiss.exe.dir/all' failed
mingw32-make[1]: *** [src/nvim/CMakeFiles/external_swiss.exe.dir/all] Error 2
```

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -5,6 +5,9 @@ configuration:
 install: []
 build_script:
 - call .ci\build.bat
+cache:
+- C:\msys64\var\cache\pacman\pkg -> .ci\build.bat
+- .deps -> third-party/CMakeLists.txt
 artifacts:
 - path: build/Neovim.zip
 - path: build/bin/nvim.exe

--- a/src/nvim/CMakeLists.txt
+++ b/src/nvim/CMakeLists.txt
@@ -299,7 +299,7 @@ if(WIN32)
   install(DIRECTORY ${PROJECT_BINARY_DIR}/windows_runtime_deps/
     DESTINATION ${CMAKE_INSTALL_BINDIR})
 
-  foreach(BIN win32yank.exe)
+  foreach(BIN cat.exe curl.exe diff.exe tee.exe tidy.exe win32yank.exe)
     unset(BIN_PATH CACHE)
     find_program(BIN_PATH ${BIN})
     if(NOT BIN_PATH)

